lib/core/src/display/shape/text/text_field/word_occurrence.rs

```diff
@@ -142,25 +142,65 @@ impl WordOccurrences {
 
 
 
-// =============
-// === Words ===
-// =============
+// ===================
+// === IndexedChar ===
+// ===================
+
+/// A character represention with the index of its occurrence in text.
+#[derive(Debug,Clone,Copy)]
+pub struct IndexedChar {
+    #[allow(missing_docs)]
+    pub index     : usize,
+    #[allow(missing_docs)]
+    pub character : char
+}
```

Comment:
```suggestion
/// A character represention with the index of its occurrence in text.
#[allow(missing_docs)]
#[derive(Debug,Clone,Copy)]
pub struct IndexedChar {
    pub index     : usize,
    pub character : char
}
```
I wonder if it will work if allow(missing_docs) would be moved before the struct (as I suggested)